sdk/src/asset_io.rs

```diff
@@ -44,6 +44,11 @@ impl CAIRead for std::fs::File {}
 impl CAIRead for std::io::Cursor<&[u8]> {}
 impl CAIRead for std::io::Cursor<Vec<u8>> {}
 
+pub trait CAIReadWrite: CAIRead + Write {}
```

Comment:
This could be in another PR later, but I'd like us to remove the `CAI` prefixes (and also not add a new `C2pa` prefix). Seems like Rust's module system should suffice where we are now using prefixes.

Reply:
Agreed, These should all be renamed, but none of them are exposed